data/chains/ropsten.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "ropsten",
+  "id": "3",
+  "contracts": { "DapiServer": "0xa70443CBE84C4e7BBcfD9DC097342932ab783e7f" },
+  "nativeToken": "ropEth",
+  "blockTime": 10,
+  "logoPath": "https://assets.coingecko.com/coins/images/13256/large/api3.jpg?1606751424"
```

Comment:
But this definitely looks like a bad logo. It should be the logo of the chain provider, right?